packages/react-menu/src/MenuDefaultOpen.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Menu, MenuTrigger, MenuList, MenuItem, MenuPopover } from './index';
+
+import { Button } from '@fluentui/react-button';
+
+export const DefaultOpen = () => (
```

Comment:
When I scroll to this story in Docs view, it is not opened anymore as I have already interacted with the page. That might confuse me.

Reply:
I'm not entirely sure how to make that clearer, this is also one of those stories where the code sample is more useful. I would err on the side of removing this example. WDYT ?

Reply:
Removed this example, not sure how to avoid this problem really